### ISSUE TITLE
CMakeLists.txt error at line 34

### ISSUE THREAD

**Issue by snaulX** [mosra/magnum-plugins#95]
CMake Error at magnum-plugins/modules/FindMagnum.cmake:328 (get_filename_component):
  get_filename_component called with incorrect number of arguments
Call Stack (most recent call first):
  magnum-plugins/CMakeLists.txt:34 (find_package)

What was must importing from Magnum in CMakeLists? And how to fix it?

**Comment by mosra** [comment]
Interesting, this is the first time I see such error.

Where do have Magnum installed? Can you upload the `CMakeCache.txt` file from the build dir?

**Comment by snaulX** [comment]
> Interesting, this is the first time I see such error.
> 
> Where do have Magnum installed? Can you upload the `CMakeCache.txt` file from the build dir?

https://pastebin.com/Ur1E9VUF - CMakeCache.txt

**Comment by mosra** [comment]
Ah, spaces in the path :) 

I'll see what all places need to be fixed to make this work. Quick workaround is to not have directory names with spaces ;)

**Comment by snaulX** [comment]
I restructured project (https://github.com/world-of-legends/Alpha_Engine/tree/main/magnum) and now all works. But if someone has same issue just reopen this issue.

**Comment by mosra** [comment]
Yeah I plan on fixing this, when I get the time. So keeping it open :)

And subscribing to the project, haha :eyes:
